test/server/cards/02_SHD/events/MysticReflection.spec.ts

```diff
@@ -0,0 +1,54 @@
+describe('Mystic Reflection', function() {
+    integration(function(contextRef) {
+        describe('Mystic Reflection\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['mystic-reflection'],
+                        groundArena: ['yoda#old-master'],
+                    },
+                    player2: {
+                        groundArena: ['specforce-soldier'],
+                        spaceArena: ['green-squadron-awing'],
+                    }
+                });
+            });
+
+            it('should give -2/-2 to an enemy unit while controlling a force unit', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.mysticReflection);
+                expect(context.player1).toBeAbleToSelectExactly([context.specforceSoldier, context.greenSquadronAwing]);
+                context.player1.clickCard(context.greenSquadronAwing);
+                expect(context.greenSquadronAwing.getPower()).toBe(0);
```

Comment:
Can we update this to a unit with more than 2 attack so we can see that it is reducing properly? Maybe restored ARC over A-Wing

Reply:
fixed with an a-wing's attack